playbooks/templates/.github/workflows/ansible-tox.yml

```diff
@@ -0,0 +1,80 @@
+---
+name: Ansible tox
```

Comment:
I think "QEMU/KVM Integration tests" is a more appropriate name, and `qemu-kvm-integration-tests.yml` for the file name

Reply:
Ah, and treat `tox` as an implementation detail? Sure. I was thinking to keep it generic in case we also start running bootc podman tests through it, but likely these will be followed by a QEMU test as well. Renamed, and I'll also update the sudo/cockpit PRs.